src/epic.rs

```diff
@@ -80,6 +80,224 @@ pub static EPICS: &[Epic] = &[
         title: "Catalogue of Women",
         wiki: Wiki::Same,
     },
+    Epic {
+        title: "Cypria",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Aethiopis",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Little Iliad",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Iliupersis",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Nostoi",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Telegony",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic Cycle",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Oedipodea",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Thebaid",
+        wiki: Wiki::Name("Thebaid (Greek poem)"),
+    },
+    Epic {
+        title: "Epigoni",
+        wiki: Wiki::Name("Epigoni (epic)"),
+    },
+    Epic {
+        title: "Alcmeonis",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Theban Cycle",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Aegimius",
+        wiki: Wiki::Name("Aegimius (poem)"),
+    },
+    Epic {
+        title: "Astronomia",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Descent of Perithous",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Idaean Dactyls",
+        wiki: Wiki::Name("Idaean Dactyls (poem)"),
+    },
+    Epic {
+        title: "Megala Erga",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Megalai Ehoiai",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Melampodia",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Wedding of Ceyx",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Capture of Oechalia",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Phocais",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Titanomachy",
+        wiki: Wiki::Name("Titanomachy (epic poem)"),
+    },
+    Epic {
+        title: "Danais",
+        wiki: Wiki::Name("Danais (epic)"),
+    },
+    Epic {
+        title: "Minyas",
+        wiki: Wiki::Name("Minyas (poem)"),
+    },
+    Epic {
+        title: "Naupactia",
+        wiki: Wiki::Same,
+    },
+    // 8th century BC to 3rd century AD
+    Epic {
+        title: "Mahābhārata",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Ramayana",
+        wiki: Wiki::Same,
+    },
+    // 3rd century BC
+    Epic {
+        title: "Argonautica",
```

Comment:
I think we should also include a 1st Century AD work that is also titled Argonauatica (by Gaius Valerius Flaccus). We could link the page as `Wiki::Name("Gaius Valerius Flaccus#Writings")`. (Link to page: https://en.wikipedia.org/wiki/Gaius_Valerius_Flaccus#Writings)